assets/pages/food-drinks.html

```diff
@@ -32,13 +32,26 @@ <h2>Brunch</h2>
 						It will be a normal continental breakfast and self-service sandwich ingredients.
 
 						<h2>Dinner</h2>
-						A hot meal will be served on Saturday and Sunday evenings.
+						<p>A hot meal will be served on Saturday and Sunday evenings.</p>
 						<br>
-						There will be vegetarian and vegan options.
-						If you have other requirements, please let us know!
+						<p>
+							There will be vegetarian and vegan options.
+							If you have other requirements, please let us know!
+						</p>
+						<br>
+						<p>Preliminary menu:</p>
+						<ul>
+							<li>Saturday: Mediterranean Pasta (vegan)</li>
+							<li>Sunday: Vegetable Risotto (vegetarian, vegan on request)</li>
+						</ul>
 
 						<h2>Plates & Cutlery</h2>
-						Please bring your own plate, cutlery, and cup.
+						<p>Please bring your own plate, cutlery, and cup.</p>
+
+						<p>
+							You need these for breakfast and brunch.
```

Comment:
```suggestion
							You need these for breakfast / brunch and coffee / tea throughout the day. 
```
...or similar? Te emphasize they need also cups.